index.js

```diff
@@ -138,6 +139,10 @@ export const startEvaluate = async ({
     })
   })
 
+  // Update top measurement stations every 12 hours
+  const pgClient = await createPgClient()
+  setInterval(async () => { await updateTopMeasurementStations(pgClient) }, 1000 * 60 * 60 * 12)
```

Comment:
I am concerned about keeping a long-living client connection. What happens when the connection is reset?

I propose to open & close a new client for every update. I think we should also add `pRetry` to handle intermittent errors.

```js
// here
setInterval(() => updateTopMeasurementStations(createPgClient), 1000 * 60 * 60 * 12)

// implementation - please refactor/clean up my spaghetti mess
export const updateTopMeasurementStations = async (pgClient) => {
  try {
    await pRetry(async () => {
      const pgClient = await createPgClient()
      try {
        await pgClient.query('REFRESH MATERIALIZED VIEW top_measurement_stations_mv')
      } finally {
        pgClient.close() // or end() ?
      }
   })
  } catch (err) {
    console.log(err)
    Sentry.captureException(err)
  }
}
```